res/controllers/Reloop-Jockey-3-ME-scripts.js

```diff
@@ -0,0 +1,546 @@
+			engine.softTakeover("[EqualizerRack1_[Channel" + k + "]_Effect1]","parameter" + l,true);
+		}
+		engine.softTakeover("[Channel" + k + "]","volume",false);
+		engine.softTakeover("[Channel" + k + "]","volume",true);
+		engine.softTakeover("[Channel" + k + "]","pregain",false);
+		engine.softTakeover("[Channel" + k + "]","pregain",true);
+  } else if (control == 0x3C && value == 0x00) {
+    Jockey3ME.MixerDeck1 = 0;
+		var k = 1;
+		for (var l = 1; l <= 3; l++) {
+			engine.softTakeover("[EqualizerRack1_[Channel" + k + "]_Effect1]","parameter" + l,false);
+			engine.softTakeover("[EqualizerRack1_[Channel" + k + "]_Effect1]","parameter" + l,true);
+		}
+		engine.softTakeover("[Channel" + k + "]","volume",false);
+		engine.softTakeover("[Channel" + k + "]","volume",true);
+		engine.softTakeover("[Channel" + k + "]","pregain",false);
+		engine.softTakeover("[Channel" + k + "]","pregain",true);
+  } else if (control == 0x3F && value == 0x7F) {
+    Jockey3ME.MixerDeck2 = 1;
+		var k = 4;
+		for (var l = 1; l <= 3; l++) {
+			engine.softTakeover("[EqualizerRack1_[Channel" + k + "]_Effect1]","parameter" + l,false);
+			engine.softTakeover("[EqualizerRack1_[Channel" + k + "]_Effect1]","parameter" + l,true);
+		}
+		engine.softTakeover("[Channel" + k + "]","volume",false);
+		engine.softTakeover("[Channel" + k + "]","volume",true);
+		engine.softTakeover("[Channel" + k + "]","pregain",false);
+		engine.softTakeover("[Channel" + k + "]","pregain",true);
+  } else if (control == 0x3F && value == 0x00) {
+    Jockey3ME.MixerDeck2 = 0;
+		var k = 2;
+		for (var l = 1; l <= 3; l++) {
+			engine.softTakeover("[EqualizerRack1_[Channel" + k + "]_Effect1]","parameter" + l,false);
+			engine.softTakeover("[EqualizerRack1_[Channel" + k + "]_Effect1]","parameter" + l,true);
+		}
+		engine.softTakeover("[Channel" + k + "]","volume",false);
+		engine.softTakeover("[Channel" + k + "]","volume",true);
+		engine.softTakeover("[Channel" + k + "]","pregain",false);
+		engine.softTakeover("[Channel" + k + "]","pregain",true);
+  }
+}
+
+Jockey3ME.EQ = function (channel, control, value, status, group) {
+  var currentDeck = parseInt(group.substring(24,25));
+  if ((Jockey3ME.MixerDeck1 == 1 && currentDeck == 1) || (Jockey3ME.MixerDeck2 == 1 && currentDeck == 2)) {
+    currentDeck += 2;
+  }
+  switch (control) {
+    case 0x2E:
+      var eqKnob = 3;
+      break;
+    case 0x2F:
+      var eqKnob = 2;
+      break;
+    case 0x30:
+      var eqKnob = 1;
+      break;
+    case 0x6D:
+      var eqKnob = 3;
+      break;
+    case 0x6E:
+      var eqKnob = 2;
+      break;
+    case 0x6F:
+      var eqKnob = 1;
+      break;
+    default:
+      print("Error on EQ chosing");
+  }
+	engine.setValue("[EqualizerRack1_[Channel" + currentDeck + "]_Effect1]","parameter" + eqKnob, script.absoluteNonLin(value,0,1,4,0,127));
+}
+
+Jockey3ME.trackSearch = function (channel, control, value, status, group) {
```

Comment:
Likewise, add a comment here to make it clear that this is one of the jog wheel modes. I think it would also help to move all the functions that handle input from the jog wheel next to each other in the code.